index.js

```diff
@@ -484,7 +484,7 @@ Command.prototype.parse = function(argv) {
     })[0];
   }
 
-  if (this._execs.has(name)) {
+  if (this._execs[name]) {
```

Comment:
Previously it was the following code.

```js
if (this._execs[name] && typeof this._execs[name] !== 'function') {
```

The function check was deleted because it was a code when it was an Array.
reference: #249

```js
> a = []; a['filter']
[Function: filter]
> o = {}; o['filter']
undefined
```

Reply:
The function check is to avoid treating built-in functions like `toString` as though they are subcommands, so I recommend a revert to the full previous code.

(The problem the Set was fixing was correctly handling a built-in property that is _not_ a function, which the old code gets wrong.)

Reply:
I was just thinking something similar! Yes. Simple and elegant.